pkg/backends/locust/backend.go

```diff
@@ -15,11 +16,23 @@ import (
 	loadTestV1 "github.com/hellofresh/kangal/pkg/kubernetes/apis/loadtest/v1"
 )
 
+var (
+	defaultImageName = "locustio/locust"
+	defaultImageTag  = "latest"
+)
+
+var (
+	// ErrRequireMinOneDistributedPod spec requires 1 or more DistributedPods
+	ErrRequireMinOneDistributedPod = errors.New("LoadTest must specify 1 or more DistributedPods")
```

Comment:
```suggestion
	ErrRequireMinOneDistributedPod = errors.New("LoadTest must specify at least 1 DistributedPods")
```

Reply:
Changing this here is out of scope, this is an existing error